test/Interpreter/SDK/archive_attributes.swift

```diff
@@ -176,3 +177,13 @@ func main() {
 
 main()
 
+// Check that we only eagerly create metadata of generic classes.
+
+// CHECK-IR-LABEL: define {{.*}} @_swift_eager_class_initialization
```

Comment:
I'd be happier if you had a positive check here as well, that all the *other* classes have static symbols emitted for them.

Reply:
It's actually and exact check of what classes should be included here.